### PR TITLE
IODEMO: Run in uniq folder

### PR DESCRIPTION
**Pull request by avildema**
iodemo test can run on different PR

### CODE REVIEW

File: buildlib/io_demo/io-demo.yml [hunk 1]
```diff
@@ -36,10 +36,20 @@ jobs:
...
           make install
-          echo "##vso[task.setvariable variable=workspace;isOutput=true]$(Build.Repository.LocalPath)/buildlib"
-          echo "##vso[task.setvariable variable=io_demo_exe;isOutput=true]$(Build.Repository.LocalPath)/install/bin/io_demo"
         displayName: Build
         name: build
+      - task: CopyFiles@2
+        inputs:
+          sourceFolder: '$(Build.Repository.LocalPath)'
+          contents: |
+            buildlib/**
```
Comment (yosefe) [inline buildlib/io_demo/io-demo.yml:45]: is it really needed?

File: buildlib/io_demo/io-demo.yml [hunk 2]
```diff
@@ -36,10 +36,20 @@ jobs:
...
+            install/bin/io_demo
```
Comment (yosefe) [inline buildlib/io_demo/io-demo.yml:46]: need all installed folder including UCX libs

File: buildlib/io_demo/io-demo.yml (hunk 1 above)
Comment (avildema) [reply buildlib/io_demo/io-demo.yml:45]: Yes, place with currapter , helper and etc scripts for work, can copy only needed script

File: buildlib/io_demo/io-demo.yml (hunk 2 above)
Comment (avildema) [reply buildlib/io_demo/io-demo.yml:46]: Ok

**Review by yosefe — APPROVED** [review]

**Comment by yosefe** [comment]
@avildema pls squash

**Pull request merged**